public/javascripts/SVLabel/src/SVLabel/canvas/ContextMenu.js

```diff
@@ -249,6 +259,15 @@ function ContextMenu (uiContextMenu) {
         });
     }
 
+    function removeLabelAndUpdateUI(labelName, labelTags) {
+        $tags.each((index, tag) => {if (tag.innerText == labelName) {tag.style.backgroundColor = "white"; } });
```

Comment:
And it looks like all the code mostly uses the tag ids instead of the tag names. So maybe it would be best to just find the tag ids and the beginning of the function and use those instead of doing more complicated things to get the text for the tags. You may even just want to have this function take in tag ids instead of a tag name.